authenticatorbridge/src/main/java/com/bitwarden/authenticatorbridge/provider/SymmetricKeyStorageProvider.kt

```diff
@@ -0,0 +1,11 @@
+package com.bitwarden.authenticatorbridge.provider
+
+import com.bitwarden.authenticatorbridge.model.SymmetricEncryptionKeyData
+
+/**
+ * TODO: document me
```

Comment:
Wanna document this

Reply:
Yes. ty :) 067fda133c5d96370a4cec544f168986e4cf765f